libs/dal/src/repositories/message/message.repository.ts

```diff
@@ -208,52 +208,6 @@ export class MessageRepository extends BaseRepository<MessageDBModel, MessageEnt
     ]);
   }
 
-  async getFeed(
-    environmentId: string,
-    query: { channels?: ChannelTypeEnum[]; templates?: string[]; emails?: string[]; _subscriberId?: string } = {},
-    skip = 0,
-    limit = 10
-  ) {
```

Comment:
Removed this one because it is not in use and can confuse other users of the repo.